docs/en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -114,6 +114,121 @@ Use the following estimates to decide how you will run your file migration:
 
 Your exact experience will vary based on your host server, the size of your files and other conditions. If your site is hosted on a managed environement (e.g.: [Common Web Platform](https://www.cwp.govt.nz/service-desk) or [SilverStripe Platform](https://docs.platform.silverstripe.com/support/)), you may not have access to the command line to manually run the migration task. Contact your hosting provider's helpdesk if that's your case.
 
+
+## Natural path vs Hash Path
+
+Two types of paths are used by SilverStripe CMS when serving files:
+* Natural paths and
+* Hash paths.
+
+Natural paths will match the path displayed in SilverStripe's asset management section (e.g.: `assets/Folder/form.pdf`). Hash paths will include a partial hash as directory name (e.g.: `assets/Folder/282318025b/form.pdf`). The hash is generated by computing the sha1 hash of the provided file and keeping the first ten characters.
+
+The hash path has the advantage of being unique for every version of the file your users might upload. This allows multiple versions of the same file to co-exist, enabling features like [file archives](04_File_Storage#archived). This is why hash path is the default file format for the protected file store.
+
+The Natural path has the advantage of being easier to understand and permanent. When a newer version of a file is published to a natural path, it will be accessible at the same URL as the previous version. This is why natural path is the default file format for the public file store.
+
+[Learn more about SilverStripe File Store](04_File_Storage)
+
+## Migration from SilverStripe 3 to Silverstripe 4.4 or later
+
+While the SilverStripe 3 file paths may look similar to the natural paths in SilverStripe 4.4, the database entries for each file have to be normalise to conform to the new SilverStripe 4 structure. File variants (e.g.: image thumbnails) are stored differently in SilverStripe 4. They will automatically be moved to their new expected location.
+
+SilverStripe 3 did not support draft file. So all migrated SilverStripe 3 files will automatically be published by the File Migration Task.
+
+SilverStripe 4 has different naming convention. File names can not contain "double underscores" in SilverStripe 4 (`__`) because those characters are used to indicate a "file variant". SilverStripe 3 files not conforming to this rule will automatically be renamed by the file migration task.
```

Comment:
```suggestion
File names can not contain "double underscores" in SilverStripe 4 (`__`) because those characters are used to indicate a "file variant". SilverStripe 3 files not conforming to this rule will automatically be renamed by the file migration task.
```

> SilverStripe 3 files not conforming to this rule will automatically be renamed by the file migration task.

To what?

Reply:
I added a complement of info